src/Microsoft.AspNetCore.SignalR/HubConnectionList.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubConnectionList : IReadOnlyCollection<HubConnectionContext>
```

Comment:
I'm just saying that I could have reused ConnectionList if I made it generic and made an interface for things that have connection ids.